PersonBook/PersonBook/PersonBook/Program.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+
+namespace PersonBook
+{
+    using Entity;
+    using CustomExceptions;
+    using Service;
+
+    class Program
+    {
+        
+        static void Main(string[] args)
+        {
+            var personService = new PersonService();
+            var isContinued = true;
+
+            while (isContinued)
+            {
+                PrintOptions();
+                string option = Console.ReadLine();
+
+                switch (option)
+                {
+                    case "1":
```

Comment:
You should wrap items in cases like below.
```
private const string Show = "1";
private const string Add = "2";
.....
case Show:
    .....
    break;
case Add:
   ....
   break;
````
```

Reply:
Done